### PR TITLE
Add container option

### PR DESCRIPTION
**Pull request by timmorey**
Proposed solution for https://github.com/sir-dunxalot/ember-tooltips/issues/304

Adds a `container` attribute to the `ember-tooltip` component, which is passed down as [tooltip.js' container option](https://popper.js.org/tooltip-documentation.html#new_Tooltip_new).  It defaults to `false`, which will render the tooltip as a sibling of its target.  Optionally, it can be set to an element or selector that identifies an alternate container for the tooltip.

In this changeset, the `APP.rootElement` config property is ignored - it used to be used to populate the container.  The previous behavior prevented apps that have this config property set from being able to render tooltips in-place.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,6 +150,20 @@ Adds a class to any tooltip:
 {{ember-tooltip class='tooltip-warning'}}
 ```
 
+#### Container
+
+| Type | HTMLElement \| String \| false |
```
Comment (maxfierke) [inline README.md]: I think these backslashes are messing with the formatting of the headers in the rendered markdown

Comment (maxfierke) [reply README.md]: Oooooooo hah no, I was just being bad at reading. That makes sense. Making them appear as code blocks would be more clear though.

**Comment by timmorey** [comment]
@maxfierke any idea what is causing the CI failure: https://travis-ci.org/sir-dunxalot/ember-tooltips/jobs/451110952#L610?  I seem to be getting occasional/inconsistent failures from that test, and it appears to be unrelated to these changes.

**Comment by maxfierke** [comment]
@timmorey that's #305. I'm hoping to take a look at it some time this week. It's flaky and should pass eventually :/

**Pull request merged**